mix.exs

```diff
@@ -65,14 +65,14 @@ defmodule TestcontainersElixir.MixProject do
       # docker api
       {:ex_docker_engine_api, "~> 1.43"},
       # ecto module
-      {:ecto_sql, "~> 3.10", optional: true},
-      {:ecto, "~> 3.10", optional: true},
+      {:ecto_sql, "~> 3.3", optional: true},
```

Comment:
interesting that we needed to downgrade some of the dependencies. By the way I have an issue on moving the ecto stuff into an another package. So that you can add testcontainers without having this weird warning in the logs saying ecto is not found.